invenio_rdm_records/fixtures/vocabularies.py

```diff
@@ -192,7 +192,8 @@ def load(self, reload=None):
             for v in VocabularyScheme.query.options(load_only("id", "parent_id")).all()
         ]
         self._loaded_vocabularies = set(v_type_ids + v_subtype_ids)
-        if reload:
+        # If it's not already loaded it means it's a new one
+        if reload and reload in self._loaded_vocabularies:
```

Comment:
is reload a key or a vocab object? it is a bit not clear from naming, looks like boolean

Reply:
It's the vocab key, simply a string, for instance `experiments` or `subjects.FOS`

Reply:
would it be difficult to rename it to indicate what it is ?

Reply:
We could do it, but I guess it´s not a patch anymore but a breaking change because we change the signature of the func. I don´t see it being used around except from in this module, but I guess we cannot know if that´s the case.

Let me know if I should change it anyway and if I should do a major release in that case.

Reply:
maybe just add a comment in the code? otherwise it looks like we are checking if bool value is in self._loaded_vocabularies, I was a bit confused trying to understand it